backend/routers/admin_analytics.py

```
@@ -266,26 +296,12 @@ async def get_ai_insights(request: Request, db=Depends(get_db)):
-                    "type": "pricing_optimization",
-                    "title": "விலை உகந்தமயமாக்கல்",
-                    "description": "புதிய கிரெடிட் தொகுப்புகளுக்கு 10% தள்ளுபடி வழங்குங்கள்.",
-                    "impact": "high",
-                    "estimated_revenue_increase": 20000
-                }
-            ],
-            "ab_tests": [],
-            "market_analysis": ["தரவு ஏற்ற முடியவில்லை"],
-            "last_updated": datetime.now().isoformat()
-        }
+        raise HTTPException(status_code=500, detail=f"Internal Server Error: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)